spec/requests/bimi_logo_spec.rb

```diff
@@ -0,0 +1,69 @@
+require 'rails_helper'
+
+RSpec.describe 'BIMI logo' do
+  it 'is available' do
+    # If you're troubleshooting this spec, there's a good chance you're trying to remove a file that
+    # appears to be unused. This comment is here to assure you that it is in-fact used, referenced
+    # as part of the BIMI DMARC records associated with the Login.gov domain. The image should not
+    # be removed as long as it's referenced by those records.
+    get '/images/login-icon-bimi.svg'
+
+    expect(response.status).to eq(200)
+  end
+
+  describe 'validity' do
+    # Test cases in this block reference best practices documentation from BIMI group:
+    # See: https://bimigroup.org/creating-bimi-svg-logo-files/
+
+    let(:path) { Rails.public_path.join('images/login-icon-bimi.svg') }
```

Comment:
Is it possible to read this from `response.body` instead of from the file system?

Reply:
> Is it possible to read this from `response.body` instead of from the file system?

Yeah, we can do that. Updated in 2a01771.